PatternPal/PatternPal.Core/Checks/ParameterCheck.cs

```diff
@@ -1,9 +1,21 @@
-﻿namespace PatternPal.Core.Checks;
+﻿using System.Linq;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
 
+namespace PatternPal.Core.Checks;
+
+/// <summary>
```

Comment:
Use <see .../> here to refer to the list of typechecks used. Look at other Checks for example comments